geode-for-redis/src/main/java/org/apache/geode/redis/internal/data/collections/SizeableObjectOpenCustomHashSet.java

```diff
@@ -62,6 +62,14 @@ public boolean remove(Object k) {
     return removed;
   }
 
+  public K getKey(final int pos) {
```

Comment:
This method name would be more accurately "getElementForIndex()" or "getFromBackingArray()" or something similar, since a Set doesn't really have a concept of keys.